src/core/input_device.c

```diff
@@ -14,6 +14,7 @@
 #include <minIni.h>
 
 #ifdef EMULATOR_BUILD
+#include "SDLaccess.h"
 #include <SDL2/SDL.h>
```

Comment:
Don't need to include this header since its already included via sdlaccess.h

Reply:
done